<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">

  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">

      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1">
        <item>
         <widget class="QPushButton" name="createBtn">
          <property name="text">
           <string>create</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="orderUserCreate">
          <property name="minimum">
           <number>1000</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orderBaseCreate">
          <item>
           <property name="text">
            <string>BTC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ETH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SOL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XRP</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orderQuoteCreate">
          <item>
           <property name="text">
            <string>USDT</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orderSideCreate">
          <item>
           <property name="text">
            <string>BUY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SELL</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="orderSizeCreate">
          <property name="minimum">
           <double>0.01</double>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="orderPriceCreate">
          <property name="minimum">
           <double>0.01</double>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orderIsProcessedCreate">
          <item>
           <property name="text">
            <string>True</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>False</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="orderOpenedAtCreate"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="getBtn">
          <property name="text">
           <string>get</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="orderIdGet">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="updateBtn">
          <property name="text">
           <string>update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="orderIdUpdate">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="orderUserUpdate">
          <property name="minimum">
           <number>1000</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orderBaseUpdate">
          <item>
           <property name="text">
            <string>BTC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ETH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SOL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XRP</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orderQuoteUpdate">
          <item>
           <property name="text">
            <string>USDT</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orderSideUpdate">
          <item>
           <property name="text">
            <string>BUY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SELL</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="orderSizeUpdate">
          <property name="minimum">
           <double>0.01</double>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="orderPriceUpdate">
          <property name="minimum">
           <double>0.01</double>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orderIsProcessedUpdate">
          <item>
           <property name="text">
            <string>True</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>False</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="orderOpenedAtUpdate"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="deleteBtn">
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="orderIdDelete">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="listBtn">
          <property name="text">
           <string>list</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>

     </layout>
    </item>
   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>26</height>
    </rect>
   </property>
  </widget>

  <widget class="QStatusBar" name="statusbar"/>

 </widget>
 <resources/>
 <connections/>
</ui>
